docs/tutorials/python/profiler.md

```diff
@@ -278,9 +275,15 @@ c = b.bind(mx.cpu(), {'a': inp})
 y = c.forward()
 mx.nd.waitall()
 profiler.set_state('stop')
+print(profiler.dumps())
```

Comment:
After profiler.dumps(), doesn't profiler reset the data?

Reply:
No, `dumps()` will only reset aggregate stats on if you specify `reset = True`. Either way, the events dumped to the file will not be affected by `dumps()`

Reply:
Yeah I meant aggregate stats. This is not a problem, I just confirmed by default reset=False for dumps().